db/schema/patch-010-009.sql

```diff
@@ -0,0 +1,97 @@
+
+-- unlockAccount v1
+CREATE PROCEDURE `unlockAccount_1` (
+    IN `inUid` BINARY(16)
+)
+BEGIN
+    DECLARE EXIT HANDLER FOR SQLEXCEPTION
+    BEGIN
+        -- ERROR
+        ROLLBACK;
+        RESIGNAL;
+    END;
+
+    START TRANSACTION;
+
+    UPDATE accounts SET lockedAt = null WHERE uid = inUid;
+    DELETE FROM accountUnlockCodes WHERE uid = inUid;
+
+    COMMIT;
+END;
```

Comment:
Is re-creating these a hangover from #35?  AFAICT the 9-10 patch doesn't drop the old ones.